src/ImageSharp/Formats/Jpeg/Components/ColorConverters/JpegColorConverter.CmykArm64.cs

```diff
@@ -0,0 +1,96 @@
+            ref Vector128<float> c3Base =
+                ref Unsafe.As<float, Vector128<float>>(ref MemoryMarshal.GetReference(values.Component3));
+
+            // Used for the color conversion
+            var scale = Vector128.Create(1 / (this.MaximumValue * this.MaximumValue));
+
+            nint n = (nint)(uint)values.Component0.Length / Vector128<float>.Count;
+            for (nint i = 0; i < n; i++)
+            {
+                ref Vector128<float> c = ref Unsafe.Add(ref c0Base, i);
+                ref Vector128<float> m = ref Unsafe.Add(ref c1Base, i);
+                ref Vector128<float> y = ref Unsafe.Add(ref c2Base, i);
+                Vector128<float> k = Unsafe.Add(ref c3Base, i);
+
+                k = AdvSimd.Multiply(k, scale);
+                c = AdvSimd.Multiply(c, k);
+                m = AdvSimd.Multiply(m, k);
+                y = AdvSimd.Multiply(y, k);
+            }
+        }
+
+        /// <inheritdoc/>
+        public override void ConvertFromRgb(in ComponentValues values, Span<float> rLane, Span<float> gLane, Span<float> bLane)
+            => ConvertFromRgb(in values, this.MaximumValue, rLane, gLane, bLane);
+
+        public static void ConvertFromRgb(in ComponentValues values, float maxValue, Span<float> rLane, Span<float> gLane, Span<float> bLane)
+        {
+            ref Vector128<float> destC =
+                ref Unsafe.As<float, Vector128<float>>(ref MemoryMarshal.GetReference(values.Component0));
+            ref Vector128<float> destM =
+                ref Unsafe.As<float, Vector128<float>>(ref MemoryMarshal.GetReference(values.Component1));
+            ref Vector128<float> destY =
+                ref Unsafe.As<float, Vector128<float>>(ref MemoryMarshal.GetReference(values.Component2));
+            ref Vector128<float> destK =
+                ref Unsafe.As<float, Vector128<float>>(ref MemoryMarshal.GetReference(values.Component3));
+
+            ref Vector128<float> srcR =
+                ref Unsafe.As<float, Vector128<float>>(ref MemoryMarshal.GetReference(rLane));
+            ref Vector128<float> srcG =
+                ref Unsafe.As<float, Vector128<float>>(ref MemoryMarshal.GetReference(gLane));
+            ref Vector128<float> srcB =
+                ref Unsafe.As<float, Vector128<float>>(ref MemoryMarshal.GetReference(bLane));
+
+            var scale = Vector128.Create(maxValue);
+
+            nint n = (nint)(uint)values.Component0.Length / Vector128<float>.Count;
+            for (nint i = 0; i < n; i++)
+            {
+                Vector128<float> ctmp = AdvSimd.Subtract(scale, Unsafe.Add(ref srcR, i));
+                Vector128<float> mtmp = AdvSimd.Subtract(scale, Unsafe.Add(ref srcG, i));
+                Vector128<float> ytmp = AdvSimd.Subtract(scale, Unsafe.Add(ref srcB, i));
+                Vector128<float> ktmp = AdvSimd.Min(ctmp, AdvSimd.Min(mtmp, ytmp));
+
+                Vector128<float> kMask = AdvSimd.CompareEqual(ktmp, scale);
```

Comment:
The AVX version has `Avx.CompareNotEqual` here, could that be the problem?

Reply:
Good catch. I will have a Look. Can not remember changing it while porting 🫣

Reply:
I tried debugging the code on VS 2022 but it just hangs when I step to the next line :(

This line should not make any difference in my eyes. Because it is ConvertFromRgb. The code which we need to look into is ConvertToRgb? Or am I wrong?

Reply:
`ConvertFromRgb` is used during encoding from ImageSharp to actual jpeg image.
`ConvertToRgb` is used during decding from jpeg to ImageSharp.
Problem can be in any of them.

Reply:
The tests work now :) Any chance to get the test output from the test run?

I changed it from Negate to Not. That makes sense :)

Reply:
I ran the tests. The images look good now

Reply:
I can confirm, the image looks good now.